src/__tests__/RenderHomePage.test.js

```diff
@@ -2,31 +2,12 @@ import RenderHomePage from "../components/pages/Home/RenderHomePage";
 import React from "react";
 import ReactDOM from "react-dom";
 import { Provider } from "react-redux";
-import { BrowserRouter as Router } from "react-router-dom";
 import { render, fireEvent } from "@testing-library/react";
 import configureStore from "redux-mock-store";
 import { toggleDrawer } from "../state/actions";
 
 const mockStore = configureStore([]);
 describe("<RenderHomePage /> test suite", () => {
-  //COMMENTING OUT DUE TO NOT HAVING LOGIN/AUTHENTICATION IN RELEASE 1--KEEPING IF NEEDED IN RELEASE 2
```

Comment:
Thanks for catching this! No one responded to my Q about it so I didn't get back to it